### PR TITLE
[Experimental] Auto-compile newsletter.

### LINKED ISSUE #110: Auto-compile newsletter
Create script which will auto-compile newsletter.

### PR DESCRIPTION
**Pull request by digitronik**
- create venv
- pip install -Ur requirements-dev.txt

- Following command will help to create newsletter with raw data. It will create file as per argument then we need to just sort and update comments manually.

```
❯ python pypune.py newsletter --help
Usage: pypune.py newsletter [OPTIONS]

  Create newsletter event.

Options:
  -m, --month [January|February|March|April|May|June|July|August|September|October|November|December]
                                  Event Month; default will take current.
  -y, --year INTEGER              Event Year; default will take current.
  --help                          Show this message and exit.

```

@raukadah @amolkahat mostly you guys compile our newsletter. take a look and let me know.

**Note**: Its experimental so after some duration we will remove previous template and update README.

fixes #110

### CODE REVIEW

**Comment by bhavin192** [comment]
Hey @digitronik we already are moving this content as a Hugo site. Once we merge that change, we can merge this one. It will be better to have this utility as `utils/<some name here>/<all the files of this PR>`.

File: utils/newsletter.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        compiled_template = TEMPLATE.format(
+            month=month, year=year, news=news, contributers=contributers
+        )
+    else:
+        click.echo(
+            "Newsletter Not Found: Confirm newsletter issue open with label:newsletter."
+        )
+        exit(1)
+
+    event_dir = PROJECT_PATH / str(year) / month
```
Comment (raukadah) [inline utils/newsletter.py:89]: PROJECT_PATH / str(year) / month needs to be changed to contents/ str(year) / month

File: utils/newsletter.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import calendar
+from datetime import datetime
+from pathlib import Path
+
+import click
+import requests
+
+TEMPLATE = """
+# PythonPune Community Newsletter [{month}-{year} Edition]
```
Comment (raukadah) [inline utils/newsletter.py:9]: Please move template to a separate file and can we add these blog tag also:
https://raw.githubusercontent.com/pythonpune/meetup-talks/master/content/2020/June/community_news.md

---
title: "PythonPune Community Newsletter [June 2020 Edition]"
date: 2020-06-28T10:51:56+05:30
url: newsletter/june-2020
author: Chandan Kumar
featured_image:
categories:
- Newsletter
tags:
---

Rest looks good

**Review by raukadah — COMMENTED** [review]
Looks nice, does most of the jobs. Few problems I found to identify which one is blog, new projects and releases.

**Comment by Pradhvan** [comment]
This looks intresting. 👍🏾

**Pull request closed without merging**